cmd/storage/cli.go

```diff
@@ -19,17 +19,17 @@ func treeAction(c *cli.Context) error {
 	}
 	owner, name, rev, path := args[0], args[1], args[2], args[3]
 
-	s, err := storage.NewLocalStorage(c.GlobalString(cmd.FlagRoot))
+	stor, err := storage.NewLocalStorage(c.GlobalString(cmd.FlagRoot))
```

Comment:
I don't really see a reason to rename this.
```suggestion
	s, err := storage.NewLocalStorage(c.GlobalString(cmd.FlagRoot))
```